Vikta-API-testing-sample/src/test/resources/test.properties

```diff
@@ -3,3 +3,8 @@ port=5054
 basePath=/api/v1/
 categoryPath=category
 categoriesPath=${categoryPath}/list
+userPath=user
+usersListPath=${userPath}/list
+userLoginPath=${userPath}/login
+userSearchPath=${userPath}/search
```

Comment:
please do the same for category and rework the category test to use it

Reply:
It's done but at main branch

Reply:
ok